### PR TITLE
Pipeline: add grafana panel

### PR DESCRIPTION
**Pull request by SeaRise**
### What problem does this PR solve?

Issue Number: ref #6518 

Problem Summary:

### What is changed and how it works?
m3CfsUHdFn

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Flash/Pipeline/Schedule/ThreadPool/TaskThreadPoolMetrics.cpp [hunk 1]
```diff
@@ -122,29 +120,6 @@ void TaskThreadPoolMetrics<is_cpu>::decThreadCnt()
     DEC_METRIC(task_thread_pool_size, 1);
 }
 
-template <bool is_cpu>
-void TaskThreadPoolMetrics<is_cpu>::updateTaskMaxtimeOnRound(uint64_t max_execution_time_ns)
```
Comment (SeaRise) [inline dbms/src/Flash/Pipeline/Schedule/ThreadPool/TaskThreadPoolMetrics.cpp:126]: feel useless

**Review by gengliqi — APPROVED** [review]

File: dbms/src/Flash/Pipeline/Schedule/ThreadPool/TaskThreadPoolMetrics.cpp (hunk 1 above)
Comment (SeaRise) [reply dbms/src/Flash/Pipeline/Schedule/ThreadPool/TaskThreadPoolMetrics.cpp:126]: replace by `tiflash_pipeline_task_execute_max_time_seconds_per_round`

**Review by windtalker — APPROVED** [review]
LGTM

**Review by windtalker — APPROVED** [review]
LGTM

**Pull request merged**